docs/architecture/deep-dives/passkeys/credentials.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 2
+---
+
+# Credentials
+
+This page provides an in-depth look at the FIDO2 credential.
+
+## What is a Credential?
+
+In general, FIDO2 works by using public/private key pairs that are generated by a FIDO2
+authenticator. The private key is protected by the authenticator, and the public key is stored
+server-side by the Relying Party (RP). The authenticator uses the private key to sign challenges
+from a RP, and the RP uses the public key to verify the signature.
+
+According to the specification, a credential is data one entity presents to another in order to
+authenticate the former to the latter. What the term "credential" refers to is generally determined
+by context, and can be one of the following:
+
+<dl>
```

Comment:
💭 Would a table or list flow better here? More native Markdown too, and you use lists in other places. Same comment for all uses of this here.

Reply:
Definition lists are semi-standard in markdown (e.g. https://www.markdownguide.org/extended-syntax/#definition-lists), but are not yet natively support by docusaurus. I think definition list is the best semantic fit here, so I would avoid tables and unordered lists if possible :) 

I also doubted using HTML, but then I found that the original markdown "specs" actually say to just use HTML when a syntax is not supported:

> For any markup that is not covered by Markdown’s syntax, you simply use HTML itself. There’s no need to preface it or delimit it to indicate that you’re switching from Markdown to HTML; you just use the tags.
> -- https://daringfireball.net/projects/markdown/syntax

I can also mention that I tried to install a plugin for definition lists ([plugin here](https://docusaurus.community/knowledge/markdown/definition-lists/)). The problem is that prettier doesn't support plugins, even if it seems like it supports unified plugins under the hood ([prettier setup here](https://github.com/prettier/prettier/blob/main/src/language-markdown/parser-markdown.js)). There is also a feature request here: https://docusaurus.io/feature-requests/p/deflist-definition-list